unit_testing/unit_test.py

```diff
@@ -1662,7 +1662,7 @@
 
     check1 = type(nemo_profiles) == coast.profile.Profile
```

Comment:
``check1 = type(nemo_profiles) == coast.profile.Profile``
--> ``check1 = type(nemo_profiles) == coast.Profile``

Since:
```
  Input In [23] in <cell line: 1>
    type(processed) == coast.profile.Profile

AttributeError: module 'coast' has no attribute 'profile'
```

Reply:
Ran with no problems.